### ISSUE TITLE
stuck in Triangle(4,5,5)

### ISSUE THREAD

**Issue by videlec** [flatsurf/flatsurf#136]
```
from pyflatsurf import flatsurf
import flatsurf as sage_flatsurf
from flatsurf.geometry.pyflatsurf_conversion import to_pyflatsurf
T = sage_flatsurf.polygons.triangle(4, 5, 5)
S = sage_flatsurf.similarity_surfaces.billiard(T)
S = S.minimal_cover(cover_type="translation")
surface = to_pyflatsurf(S)
for connection in surface.saddle_connections(flatsurf.Bound(5, 0)):
    v = connection.vector()
    print(v)
    decomposition = flatsurf.makeFlowDecomposition(surface, v)
    decomposition.decompose()
```
hangs after
```
(1, 0)
((1/2*c^4 - 2*c^2 + 2 ~ 1.6234898), (1/2*c^3 - 3/2*c ~ 0.78183148))
((1/2*c^4 - 3/2*c^2 + 1 ~ 2.5244587), (-1/2*c^5 + 3*c^3 - 4*c ~ 0.34794774))
```
Most probably similar to #140 since when we manually set the bound to `100` it finds properly the decomposition
```
....: for connection in surface.saddle_connections(flatsurf.Bound(5, 0)):
....:     v = connection.vector()
....:     print(v)
....:     decomposition = flatsurf.makeFlowDecomposition(surface, v)
....:     print([bool(comp.cylinder() == True or comp.cylinder() == False) for comp in decomposition.components()])    
....:     ans = decomposition.decompose(100r)
....:     print([bool(comp.cylinder() == True or comp.cylinder() == False) for comp in decomposition.components()])
....:     print()
```
gives
```
(1, 0)
[False, False]
[True, True, True, True, True, True]

((1/2*c^4 - 2*c^2 + 2 ~ 1.6234898), (1/2*c^3 - 3/2*c ~ 0.78183148))
[False, False]
[True, True, True, True, True, True]

((1/2*c^4 - 3/2*c^2 + 1 ~ 2.5244587), (-1/2*c^5 + 3*c^3 - 4*c ~ 0.34794774))
[False]
[True, True]
```

**Comment by saraedum** [comment]
Is this really a bug? Isn't these one of the cases where our approach simply doesn't work because intervalxt will loop forever?

When I pass a `limit` to the `decomposeFlowDecomposition` call it does not hang.

**Comment by videlec** [comment]
It is a bug: if you pass a big enough limit the result is a decomposition without any undetermined component.

**Comment by videlec** [comment]
fixed in release 1.1.2
